inventory

```diff
@@ -16,6 +16,11 @@ openshift_install_examples=false
 # Domain name that will be used by the Openshift router
 openshift_master_default_subdomain=cloudapps.example.com
 
+# Image used for the kubevirt/web-ui deployment
+kubevirt_web_ui_image_name=docker.io/mareklibra/kubevirt-web-ui:v1.3-8
```

Comment:
talk to @fabiand about getting this published to docker.io/kubevirt

Reply:
Sure. This will take some time, can we fix that in a follow-up, please?